packages/web-components/src/link/link.styles.ts

```diff
@@ -0,0 +1,87 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorBrandForegroundLink,
+  colorBrandForegroundLinkPressed,
+  colorNeutralForeground2,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  ${display('inline')}
+
+  :host {
+    font-size: inherit;
+  }
+
+  :host .control {
+    background-color: transparent;
+    bo-sizing: border-box;
+    color: ${colorBrandForegroundLink};
+    cursor: pointer;
+    display: inline;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
```

Comment:
We `inherit` on the host but set it here on in the `.control` class. This isn't ideal because you can't set `font-size` directly on the implementation of `fluent-link` and will need to use a `::part` selector. Instead this should be inherit and the `font-size: ${fontSizeBase300}` should be set on the host.

Reply:
And if this is truly meant to to be inline, perhaps it should just inherit all up. This is the default behavior for an inline html anchor. Thoughts?

Reply:
In terms of the inheritance and where, etc - this is intentional I believe. 

On one hand, we want a default set (base 300), but we want to inherit given links are _typically_ used inline with text and we should inherit so that the inline nature is there. If we set the default on the host, then we cannot inherit at the control set. You'll notice that the control when an href is present should inherit. In this case, I'm actually unsure if we want to inherit everything...this is a 'design system' choice. BUT, where we inherit needs to be at the host initially...similar to above, we need to discuss a few of the choices here since I don't think it makes sense to also have a button variant of this right now.